A. Team/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace A._Team
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
+            int countOfSol = 0;
+            int count = 0;
+            int n = rd.NextInt();
+            int[, ] arr = new int[n,3];
```

Comment:
Use var to declare variables
```suggestion
            var rd = new Reader();
            var countOfSol = 0;
            var count = 0;
            var n = rd.NextInt();
            var arr = new int[n,3];
```